Source/Scene/GlobeSurfaceTileProvider.js

```diff
@@ -252,6 +253,17 @@ define([
             }
         },
 
+        /**
+         * Gets an event that's raised when an imagery layer is added, shown, hidden, moved, or removed.
+         * @memberof GlobeSurfaceTileProvider.prototype
+         * @type {Event}
+         */
+        imageryLayersUpdatedEvent : {
```

Comment:
Regarding 1 and 2, the scene watches for this event to trigger a new render, and aggregating the events in `ImageryLayerCollection` seemed cleaner, especially because we have to handle removing the callbacks if the globe changes. Also for 3, I'll try and set this up to be raised after the update completes.

Reply:
Oh sorry, my claim that the event isn't used was based on search the diff, and GitHub had helpfully hidden part of it.  Nevermind!  3 is probably worth a bit of thought, but may not be a big deal.